server/gokbg3/grails-app/services/org/gokb/TitleLookupService.groovy

```diff
@@ -556,13 +564,16 @@ class TitleLookupService {
 
               // Raise a review request
 
-              if(added) {
-                ReviewRequest.raise(
-                    the_title,
-                    "'${metadata.title}' added as a variant of '${the_title.name}'.",
-                    "Title was matched via secondary id, but had a different name.",
-                    user, project
-                    )
+              if (added) {
+                def rr = reviewRequestService.raise(
```

Comment:
There's nothing further done with `rr` - so maybe drop that assignment?
(Same for the following cases.)